packages/apollo-graphql/src/transforms.ts

```diff
@@ -101,6 +101,13 @@ function sorted<T>(
 // order, it can make sure the queries are treated as identical.
 export function sortAST(ast: DocumentNode): DocumentNode {
   return visit(ast, {
+    Document(node: DocumentNode) {
+      return {
+        ...node,
+        // Use sortBy because 'definitions' is not optional.
+        definitions: sortBy(node.definitions, "kind", "name.value")
```

Comment:
I think it would be nice to be explicit that this means that fragments go before operations.

Reply:
Resolved in 5cd843ddd354fd3a4fecd36a41829d73cb98ed0b